{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/identityorchestration\/tsidojourneyerror\/networkerror"]}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration","doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError"]]},"schemaVersion":{"minor":3,"patch":0,"major":0},"metadata":{"modules":[{"name":"IdentityOrchestration"}],"externalID":"s:21IdentityOrchestration17TSIdoJourneyErrorO07networkE0yA2CmF","symbolKind":"case","roleHeading":"Case","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"networkError","kind":"identifier"}],"title":"TSIdoJourneyError.networkError"},"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"networkError","kind":"identifier"}]}]}],"abstract":[{"type":"text","text":"Could not connect to server, or server did not respond before timeout."}],"identifier":{"url":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError\/networkError","interfaceLanguage":"swift"},"references":{"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration/TSIdoJourneyError":{"url":"\/documentation\/identityorchestration\/tsidojourneyerror","title":"TSIdoJourneyError","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"TSIdoJourneyError","kind":"identifier"}],"abstract":[{"text":"The enum for the start Journey errors.","type":"text"}],"type":"topic","identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError","kind":"symbol","navigatorTitle":[{"text":"TSIdoJourneyError","kind":"identifier"}]},"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration/TSIdoJourneyError/networkError":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}],"role":"symbol","type":"topic","identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError\/networkError","abstract":[{"text":"Could not connect to server, or server did not respond before timeout.","type":"text"}],"title":"TSIdoJourneyError.networkError","url":"\/documentation\/identityorchestration\/tsidojourneyerror\/networkerror","kind":"symbol"},"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration":{"kind":"symbol","type":"topic","abstract":[],"role":"collection","identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration","url":"\/documentation\/identityorchestration","title":"IdentityOrchestration"}}}